SphereAPIDeviceCodeAuth/README.md

```diff
@@ -0,0 +1,20 @@
+# Access Azure Sphere API using AAD Device code flow
```

Comment:
Suggest "Access Azure Sphere Public REST API"   (to differentiate from APIs on the device)